statsforecast/core.py

```diff
@@ -61,14 +61,20 @@ def __eq__(self, other):
             self.indptr, other.indptr
         )
 
-    def fit(self, models):
+    def fit(self, models, fallback_model=None):
         fm = np.full((self.n_groups, len(models)), np.nan, dtype=object)
         for i, grp in enumerate(self):
             y = grp[:, 0] if grp.ndim == 2 else grp
             X = grp[:, 1:] if (grp.ndim == 2 and grp.shape[1] > 1) else None
             for i_model, model in enumerate(models):
-                new_model = model.new()
-                fm[i, i_model] = new_model.fit(y=y, X=X)
+                try:
+                    new_model = model.new()
+                    fm[i, i_model] = new_model.fit(y=y, X=X)
+                except Exception as error:
+                    if fallback_model is not None:
+                        fm[i, i_model] = fallback_model.fit(y=y, X=X)
```

Comment:
Dont' we need to use `fallback_model.new()` here as well?

Reply:
Thanks for the comment. I addressed it.